Editor/Scripts/Startup/RegressionPackagePopup.cs

```diff
@@ -396,7 +396,8 @@ private void ImportSample(string sampleName)
             }
             catch (Exception e)
             {
-                RGDebug.LogError("Failed to import sample: " + e.Message);
+                RGDebug.LogError("Failed to import sample");
+                RGDebug.LogException(e);
```

Comment:
Since this is our own function, can't we have LogException in RGDebug take a message? Then we'd only have to do
```csharp
RGDebug.LogException("My message", e);
```
(this is a non-blocking idea)

Reply:
Added 👍  It's a good idea to prevent duplicate 'RGDebug' statements throughout the code. I added an option 'message' param in 'LogException'